nsfc-plugin/src/main/java/org/osc/controller/nsfc/entities/InspectionPortEntity.java

```diff
@@ -47,6 +47,10 @@ public String getElementId() {
         return this.elementId;
     }
 
+    public void setElementId(String elementId) {
```

Comment:
does this need to be public? Would prefer to make this private or package private if possible.